src/mono/mono/metadata/custom-attrs.c

```diff
@@ -1904,7 +1906,10 @@ mono_custom_attrs_from_index_checked (MonoImage *image, guint32 idx, gboolean ig
 		return NULL;
 	i --;
 	// initial size chosen arbitrarily, but default is 16 which is rather small
-	attr_array = g_array_sized_new (TRUE, TRUE, sizeof (guint32), 128);
+	dn_vector_custom_alloc_params_t vec_params = {0,};
+	vec_params.capacity = 128;
+	vec_params.attributes = DN_VECTOR_ATTRIBUTE_MEMORY_INIT;
```

Comment:
Not really necessary since we don't have any gaps in the data in this array and we never look past the end.